### ISSUE TITLE
NumberPad

### ISSUE THREAD

**Issue by ldrahnik** [shiecldk/ASUS-ZenBook-Pro-Duo-15-OLED-UX582-Hackintosh#17]
I am open to being involved in the changes required to use the [NumberPad driver](https://github.com/asus-linux-drivers/asus-touchpad-numpad-driver) with OS X as well.

**Comment by magnaprog** [comment]
@ldrahnik Maybe you can look into porting the code with AI tools.

**Comment by ldrahnik** [comment]
@magnaprog Thanks! Good idea. But I do not think it can be done completely without testing. Dont you have OS X?

**Comment by magnaprog** [comment]
@ldrahnik Yes I have macOS. However, I'm mostly commit to other projects recently. If you have this laptop, maybe you can look into this problem first.

**Comment by shiecldk** [comment]
@ldrahnik I have macOS. However, I'm mostly commit to other projects recently. If you have this laptop, maybe you can look into this problem first.

**Comment by shiecldk** [comment]
@ldrahnik What do you need from me to start this port?
